src/data/unicorns.json

```
@@ -121,4 +121,18 @@
     "color": "#7C4DFF",
     "roles": ["developer"]
   }
+  {
+    "id": "MDutro",
+    "name": "Micah Dutro",
+    "firstName": "Micah",
+    "lastName": "Dutro",
+    "description": "A non-profit lawyer turned budding web developer.",
+    "socials": {
+      "github": "MDutro",
```

We also want to remove the trailing comma here per JSON rules:
```suggestion
      "github": "MDutro"
```